Get result from which/where first

rosen-vladimirov opened the pull request:
Get the result from which/where output first instead of the result from npm globally installed modules.
This is required in cases where the searched package is added to PATH from different place.

rosen-vladimirov committed:
Get result from which/where first

Get the result from which/where output first instead of the result from npm globally installed modules.
This is required in cases where the searched package is added to PATH from different place.

tsvetie reviewed (approved):
Code: looks good to me.
Tests: you could reuse parts of the tests code and I am not sure if your restore works.

rosen-vladimirov commented:
@tsvetie , the restore works. We've discussed the tests with @TsvetanMilanov and I'll rewrite them in a separate PR.

rosen-vladimirov committed:
Set version to 2.0.0

Update major version as there's a huge behavior change - now we search for the executable in PATH and in case we do not find it, we try to find it in globally installed npm packages.
